nginx/default.toml

```diff
@@ -19,6 +19,12 @@ worker_processes = 4
 # worker_connections: Connections per Worker Process.  Default = 1024
 worker_connections = 1024
 
+[[include.additional_configuration_file]]
+config_file = "mime.types"
+
+[[include.additional_configuration_file]]
```

Comment:
Hi Mike,

In my reading of ["Array of Tables"](https://github.com/toml-lang/toml#user-content-array-of-tables) in the TOML spec, and the ["Add configuration to plans"](https://www.habitat.sh/docs/create-packages-configure/) section in the Habitat docs, I didn't find any alternative.